### PR TITLE
Added reference note to known bug in PowerShell modules

### PR DESCRIPTION
**Pull request by Snozzberries**
Reference issue: https://github.com/MicrosoftDocs/windowsserverdocs/issues/3492

### CODE REVIEW

**Comment by PRMerger6** [comment]
@Snozzberries : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/manage/windows-admin-center/configure/use-powershell.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ If you have a large organization with multiple Windows Admin Center servers, you
 
 ## Use PowerShell to import or export your connections (with tags)
 
+[!NOTE] With version 2103 there is a known bug with the included modules requiring modification to the header parameters shown in this [Gist](https://gist.github.com/Snozzberries/3ed0a64f691b262d2b5edcbfdf4d2a37).
```
Comment (illfated) [inline WindowsServerDocs/manage/windows-admin-center/configure/use-powershell.md:17]: Good start, but we also need the MarkDown formatting to make it work properly as a Note blob:
```suggestion
> [!NOTE]
> With version 2103, there is a known bug with the included modules, requiring modification to the header parameters shown in this [Gist](https://gist.github.com/Snozzberries/3ed0a64f691b262d2b5edcbfdf4d2a37).
```

Comment (illfated) [reply WindowsServerDocs/manage/windows-admin-center/configure/use-powershell.md:17]: @Snozzberries : Will you be able to return to this PR, to review & merge the suggested improvement?

**Comment by PRMerger14** [comment]
@Snozzberries : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/manage/windows-admin-center/configure/use-powershell.md (hunk 1 above)
Comment (Snozzberries) [reply WindowsServerDocs/manage/windows-admin-center/configure/use-powershell.md]: @illfated Sorry thought I had accepted your suggestion. New commit added.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger14** [comment]
@illfated: I'm sorry - only the author of this article, @jwwool, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. My sign-off comment is based on a previous agreement with the team.

**Comment by Snozzberries** [comment]
This is resolved in latest version.

https://techcommunity.microsoft.com/t5/windows-admin-center-blog/windows-admin-center-version-2103-2-is-now-generally-available/ba-p/2384136#toc-hId--220972258

**Pull request closed without merging**